src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/employee.yml

```diff
@@ -9,12 +9,43 @@ admin_employees_search:
   path: /
   methods: [POST]
   defaults:
-    _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Employee:search'
-    _legacy_controller: AdminEmployees
+    _controller: 'PrestaShopBundle:Admin\Common:searchGrid'
+    gridDefinitionFactoryService: prestashop.core.grid.definition.factory.employee
```

Comment:
Why this ?

Reply:
its a common search action, we need to provide it with grid definition factory so it can create filters form.

Reply:
Maybe we should write doc about, wdyt ?

Reply:
probably, but its not related to this PR.